src/main/java/org/takes/facets/auth/social/PsTwitter.java

```diff
@@ -64,57 +71,112 @@
     private final transient String key;
 
     /**
+     * Request for fetching app trequest.
+     */
+    private final transient com.jcabi.http.Request trequest;
```

Comment:
@adko-pl call it `tkn` then. take note that you changed the javadoc when doing the refactor. It should read `fetching app token`.